### ISSUE TITLE
Vertical Navigation - Slider Mode - selected element is not identified with accessibility attribute "aria-current=page"

### ISSUE THREAD

**Issue by paulaclemente** [lumada-design/hv-uikit-react#3343]
### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

Selected element is not identified with accessibility attribute `aria-current=page`

### Expected behavior 🤔

Following the best practices for accessibility, selected element should be identified with accessibility attribute `aria-current=page`

Ref: https://www.w3.org/TR/wai-aria-1.1/#aria-current

### Steps to reproduce 🕹

Steps:

1. Open Storybook
2. Go to "Vertical Navigation -> Slider Mode" example
3. Navigate to the item that is selected, right-click on it and open the Web Inspector: notice that the "aria-current" attribute is not set.

**Comment by plagoa** [comment]
Bug: https://hv-eng.atlassian.net/browse/HVUIKIT-6725

**Comment by VGervasio** [comment]
Question: shouldn't the elements contain accessible link elements?

**Comment by zettca** [comment]
Hey @VGervasio @paulaclemente.
We'll be adding link elements to the sample, make the necessary changes to the component.
